app/views/api/properties/_load_repairs_history_button.html.erb

```diff
@@ -3,7 +3,7 @@
     <% if work_orders.any? %>
       Repairs history is showing jobs raised in the <strong>last 2 years</strong>.
     <% else %>
-      There are no work orders for this <%= property_description.downcase %> within the <strong>last 2 years</strong>.
+      There are no work orders for <%= this_or_these_property_type(property_description.downcase) %> within the <strong>last 2 years</strong>.
```

Comment:
Can you please inline `this_or_these_property_type`, for readability? Also, the current method name make it look like it would handle any pluralized `property_type`, when in fact it handles only _facilities_.

Reply:
Would you suggest changing the name, or changing the method to handle all cases of plurals?

Reply:
@ayghor ^^^

Reply:
I think you should get rid of the method and just put the if statement inline on the template.